### ISSUE TITLE
Add expand and collapse for detailed steps

### ISSUE THREAD

**Issue by luisxiaomai** [MacPaw/xcsummary#2]
Hi guy,

Thanks for sharing your implementation for this html result which is almost what I wanted. 
But if all detailed steps can be collapsed just like what we see test result in xcode will be perfect.
User can choose to expand and see which screenshot.

B.R.
Luis

**Comment by Krivoblotsky** [comment]
Added to the roadmap. Thanks.

**Comment by ampilogov** [comment]
It will be amazing

**Comment by Krivoblotsky** [comment]
@ivampir, thanks for the PR. I'll test it today and merge if everything is ok;)

**Comment by ampilogov** [comment]
@Krivoblotsky  Hi! Have you test it?

**Comment by Krivoblotsky** [comment]
Sorry for the delay. 
Yes, we've took a look at it. We've found some issues and unfortunately we can't merge it for now. 
The major issue is: clicking on a row expands just the next row, but not the group. Thats a bit strange behaviour and we should avoid it. (I've attached the video to demostrate it).
The minor issue: there should be some kind of indicator to see if the row expanded or not.

[summary.mov.zip](https://github.com/MacPaw/xcsummary/files/868957/summary.mov.zip)

By the way, thanks a lot for your work, we really appreciate it!
